graphbolt/src/index_select.h

```diff
@@ -6,11 +6,23 @@
 #ifndef GRAPHBOLT_INDEX_SELECT_H_
 #define GRAPHBOLT_INDEX_SELECT_H_
 
+#include <graphbolt/fused_sampled_subgraph.h>
 #include <torch/script.h>
 
 namespace graphbolt {
 namespace ops {
 
+/** @brief Implemented in the cuda directory. */
+std::tuple<torch::Tensor, torch::Tensor> UVAIndexSelectCSCImpl(
+    torch::Tensor indptr, torch::Tensor indices, torch::Tensor index);
+
+/** @brief Implemented in the cuda directory. */
+std::tuple<torch::Tensor, torch::Tensor> IndexSelectCSCImpl(
+    torch::Tensor indptr, torch::Tensor indices, torch::Tensor index);
+
+std::tuple<torch::Tensor, torch::Tensor> IndexSelectCSC(
```

Comment:
Could you add documentations for this function?

Reply:
Added, can you recheck?